### DISCUSSION TITLE
Which CPU brand for Windows commercial license?

### DISCUSSION THREAD

**Discussion by pmaskarinec** [emgucv/emgucv#805] [Q&A] [answered]
I'd like to use Emgu for my commercial Windows application. Should I care about the CPU brand (Intel vs AMD) when the commercial version has the Intel®️ IPP optimizations? I'm not sure if Intel®️ IPP works on AMD CPUs.

**Comment by emgucv** [comment, marked as answer]
AMD CPUs will also benefit from IPP optimized binary. See these QA on the Intel IPP website:
https://community.intel.com/t5/Intel-Integrated-Performance/Can-I-use-Intel-IPP-on-AMD-CPU/td-p/898377
